packages/entities/entities-certificates/src/components/CACertificateConfigCard.vue

```diff
@@ -114,6 +121,26 @@ const issuer = computed((): string => parsedCertData.value?.schemaIssuer || '')
 const expiry = computed((): string => parsedCertData.value?.schemaExpiry ? formatUnixTimeStamp(parsedCertData.value?.schemaExpiry) : '')
 const keyUsages = computed((): string[] => parsedCertData.value?.schemaKeyUsages || [])
 
+const getPropValue = (propName: string, slotProps?: Record<string, any>) => {
```

Comment:
Can you make this method available in a composable so you don't need to duplicate it in each entity?

Reply:
Updated.